walletlibrarydemo/src/main/java/com/microsoft/walletlibrarydemo/feature/issuance/presentationlogic/SampleViewModel.kt

```diff
@@ -0,0 +1,50 @@
+package com.microsoft.walletlibrarydemo.feature.issuance.presentationlogic
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.net.Uri
+import androidx.lifecycle.ViewModel
+import com.microsoft.walletlibrary.VerifiedIdClientBuilder
+import com.microsoft.walletlibrary.requests.VerifiedIdRequest
+import com.microsoft.walletlibrary.requests.input.VerifiedIdRequestURL
+import com.microsoft.walletlibrary.verifiedid.VerifiedId
+import com.microsoft.walletlibrarydemo.db.VerifiedIdDatabase
+
+class SampleViewModel(@SuppressLint("StaticFieldLeak") val context: Context): ViewModel() {
+    var verifiedIdRequestResult: Result<VerifiedIdRequest<*>>? = null
+    var verifiedIdResult: Result<Any?>? = null
+    private val verifiedIdClient = VerifiedIdClientBuilder(context).build()
+    private val verifiedIdDao = VerifiedIdDatabase.getInstance(context).verifiedIdDao()
+
+    suspend fun initiateRequest(requestUrl: String) {
+        val verifiedIdRequestUrl =
+            VerifiedIdRequestURL(Uri.parse(requestUrl))
+        verifiedIdRequestResult = verifiedIdClient.createRequest(verifiedIdRequestUrl)
+    }
+
+    suspend fun completeIssuance() {
+        verifiedIdResult = verifiedIdRequestResult?.getOrNull()?.complete()
```

Comment:
if you unwrap it, then will not have to use getOrNull? Might be neater? Also can you add a comment or two in this class for developers? They will be looking at this code to see how they can use our SDK within their own app, so maybe over-commenting might be helpful here :)